server/protocols/webadmin/webadmin-mailbox/src/main/java/org/apache/james/webadmin/service/EventDeadLettersRedeliverService.java

```diff
@@ -37,6 +39,7 @@
     private final EventBus eventBus;
     private final EventDeadLetters deadLetters;
 
+    @Inject
     public EventDeadLettersRedeliverService(EventBus eventBus, EventDeadLetters deadLetters) {
```

Comment:
every time you read or write @Inject and public at the same time, you have a visibility problem.
Injection makes usually access to constructor useless.
If you use it for tests, say it with  @VisibleForTesting